test/schema/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -91,11 +91,11 @@ describe('v1.0.0 ==> v2.0.0', function () {
             expect(v2.item[0].request.body.raw).to.eql('something');
         });
 
-        it('should strip out all request and folder ids by default', function () {
+        it('should not strip out all request and folder ids by default', function () {
```

Comment:
This description is confusing me here. 

It says all ids should not be stripped, and there's an assertion for `.id` being stripped off.